### PR TITLE
Steps 7 and 9

### PR DESCRIPTION
**Pull request by heatherbolze**
I'm a little iffy on LoginServletTest, so be sure to take a good look at the new test I wrote and the old ones I altered, thanks!

### CODE REVIEW

File: src/main/java/codeu/controller/RegisterServlet.java [hunk 1]
```diff
@@ -52,6 +52,7 @@ public void doPost(HttpServletRequest request, HttpServletResponse response)
 
 		String username = request.getParameter("username");
 		String password = request.getParameter("password");
+		String passwordHash = BCrypt.haspw(password,BCrypt.gensalt());
```
Comment (AndresGutierrez01) [inline src/main/java/codeu/controller/RegisterServlet.java:55]: In line 69 you forgot to change the 'password' parameter to 'passwordHash'

File: src/main/java/codeu/model/data/User.java [hunk 2]
```diff
@@ -35,7 +35,7 @@
   public User(UUID id, String name, String password, Instant creation) {
```
Comment (AndresGutierrez01) [inline src/main/java/codeu/model/data/User.java:35]: You forgot to change the 'password' parameter to 'hashedPassword'

File: src/main/java/codeu/controller/RegisterServlet.java (hunk 1 above)
Comment (heatherbolze) [reply src/main/java/codeu/controller/RegisterServlet.java:55]: Yup you're right, thanks

File: src/main/java/codeu/model/data/User.java (hunk 2 above)
Comment (heatherbolze) [reply src/main/java/codeu/model/data/User.java:35]: Got it

File: src/main/java/codeu/controller/RegisterServlet.java (hunk 1 above)
Comment (draspawn) [reply src/main/java/codeu/controller/RegisterServlet.java:55]: BCrypt.haspw is not a method, should be hashpw

Also, you have not included the import statement for BCrypt in the file that is in the pull request.

File: src/main/java/codeu/controller/LoginServlet.java [hunk 3]
```diff
@@ -59,27 +59,41 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
...
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response)
       throws IOException, ServletException {
     String username = request.getParameter("username");
+    String password = request.getParameter("password");
+
+    if (userStore.isUserRegistered(username)) {
+        User user = userStore.getUser(username);
+    
+    if(BCrypt.checkpw(password,user.getPassword())) {
```
Comment (draspawn) [inline src/main/java/codeu/controller/LoginServlet.java:75]: indent of your if, else logic here makes the code hard to parse.  Review style guide and apply to the code to make it more readable.

File: src/test/java/codeu/controller/LoginServletTest.java [hunk 4]
```diff
@@ -71,30 +71,34 @@ public void testDoPost_NewUser() throws IOException, ServletException {
...
+  public void testDoPost_ExistingUser_CorrectPassword() throws IOException, ServletException {
     Mockito.when(mockRequest.getParameter("username")).thenReturn("test username");
 
     UserStore mockUserStore = Mockito.mock(UserStore.class);
     Mockito.when(mockUserStore.isUserRegistered("test username")).thenReturn(true);
     loginServlet.setUserStore(mockUserStore);
 
-    HttpSession mockSession = Mockito.mock(HttpSession.class);
-    Mockito.when(mockRequest.getSession()).thenReturn(mockSession);
+    //HttpSession mockSession = Mockito.mock(HttpSession.class);
```
Comment (draspawn) [inline src/test/java/codeu/controller/LoginServletTest.java:96]: Just curious, why did you comment out the session handling code in your tests?
